cray_product_catalog/catalog_update.py

```diff
@@ -72,6 +73,24 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def format_product_cm_name():
```

Comment:
if we need to create cm, lets make the function with i/o. Let's not use global variable. Harder to track n debug.
We are writing all restriction we are not applying all.